config/foreman.yml

```diff
@@ -6,9 +6,29 @@
   :host: 'https://localhost/'
 
   # Credentials. You'll be asked for them interactively if you leave them blank here
+  # Possible values:
+
+  # Basic Auth:
+  #default_auth_type: 'Basic_Auth'
```

Comment:
This should be `#:default_auth_type:` also below for PGF and ACF. Notice the coulmn before `default...`.

If user will just uncomment those lines, it will not work without the column (parser will not treat it like a symbol IIUC).

Reply:
good catch!